src/routes/Make.tsx

```diff
@@ -30,15 +30,35 @@ const Make: React.FC = () => {
     Object.keys(currentDrink).filter(key => key.includes('strIngredient') && ingredients.push(currentDrink[key]));
   }
 
-  console.log('spec', {data, error});
+  function the_dude(d: string) {
+    if (d === 'White Russian') {
+      return (
+        <div>
+          <img src="./images/the-dude.gif" alt="The Dude" width="200" height="auto" />
+        </div>
+      )
+    }
+  }
+
+  console.log('spec', { data, error });
+
+  function checkDrink(drink: string) {
+    if (drink === 'White Russian') {
+      return (
+        ', Dude?'
+      )
+    }
+  }
 
   return (
     <Main name='make'>
       { error && <Error/> }
       { !data && !error && <Loading/> }
       { currentDrink &&
         <Card>
-          <CardHeader title={`You want to make a ${input}?`}/>
+          <CardHeader title={`You want to make a ${input}${checkDrink(input)}`}>
+            {the_dude(input)}
```

Comment:
i'd just remove this function and put the logic inline (and remove the excess div)

```js
{ input === 'White Russian' &&
  <img src="./images/the-dude.gif" alt="The Dude" width="200" height="auto" />
}
```
```

Reply:
Also, don't use inline styles for the width and height. Put a className on the img and then use a .scss file to target it